src/dbnode/client/config.go

```diff
@@ -87,8 +87,8 @@ type Configuration struct {
 
 // ProtoConfiguration is the configuration for running with ProtoDataMode enabled.
 type ProtoConfiguration struct {
-	SchemaFilePath string `yaml:"schemaFilePath"`
-	MessageName    string `yaml:"messageName"`
+	// Whether proto is enabled.
```

Comment:
nit: Field name comments should start with the name of the field, i.e. `// Enabled specifies whether proto is enabled.`

Reply:
ok